.github/scripts/update-manifests-tags.sh

```
@@ -6,12 +6,28 @@
-    component=${varname#${prefix}}
+    [[ $varname =~ "${spec_prefix}" ]] || continue
+    component=${varname#${spec_prefix}}
     component=${component//_/-}
+    echo "  Updating branch/tag for $(basename $component) to: $value"
     update_tags $(basename $component) $value
-done
+done
+
+echo "Updating component repository organizations..."
+env | while IFS="=" read varname value; do
+    [[ $varname =~ "${org_prefix}" ]] || continue
```

Shellcheck (0.10.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)